subsystem/cluster_test.go

```diff
@@ -821,6 +821,55 @@ var _ = Describe("cluster install - DHCP", func() {
 	})
 })
 
+var _ = Describe("Create cluster with BaseDNS", func() {
+	var (
+		ctx         = context.Background()
+		clusterCIDR = "10.128.0.0/14"
+		serviceCIDR = "172.30.0.0/16"
+	)
+	AfterEach(func() {
+		clearDB()
+	})
+	createClusterWithBaseDNS := func(baseDNS string) (*installer.RegisterClusterCreated, error) {
+		return userBMClient.Installer.RegisterCluster(ctx, &installer.RegisterClusterParams{
+			NewClusterParams: &models.ClusterCreateParams{
+				BaseDNSDomain:            baseDNS,
+				ClusterNetworkCidr:       &clusterCIDR,
+				ClusterNetworkHostPrefix: 23,
+				Name:                     swag.String("test-cluster"),
+				OpenshiftVersion:         swag.String(openshiftVersion),
+				PullSecret:               swag.String(pullSecret),
+				ServiceNetworkCidr:       &serviceCIDR,
+				SSHPublicKey:             sshPublicKey,
+			},
+		})
+	}
+	It("Should thow en error with invalid BaseDNS 'example'", func() {
```

Comment:
1. Spelling: "throw an error"
2. Use loop of tests instead a copy paste of the same test logic
3. Please use more descriptive test names. for example: "Illegal character in domain name", "too short domain name" etc.

Reply:
👍🏽 Thanks for the input!